src/ajax/interceptors.cljc

```diff
@@ -189,9 +189,11 @@
 (defrecord ProcessUrlParameters []
   Interceptor
   (-process-request [_ {:keys [method] :as request}]
-    (let [if-get-reduce (if (= method "GET") reduced identity)]
-      (if-get-reduce (update request :uri
-                       (uri-with-params request)))))
+    (cond->
+      (update request :uri
+              (partial uri-with-params request))
+      (= method "GET")
```

Comment:
I don't think this does quite the same thing. It's been a while but this looks like it always marks it as reduced, while the original didn't.

Reply:
No, if `(= method "GET")` is false, it won’t be marked as reduced. `macroexpand`ing the `cond->` block gives this (slightly rewritten for clarity):

```clojure
(let* [r (update request :uri (partial uri-with-params request))] (if (= method "GET") (reduced r) r))
```
Are you ok with the `cond->` or would you prefer another way of writing this that would maybe be easier to read?

Reply:
Actually, leave it. This is just an effect of me being out of practice with Clojure. I'm not going to slowly rewrite it to suit my current facilty!